students/colephalen/lesson05/assignment/src/database.py

```diff
@@ -0,0 +1,88 @@
+import csv
+from pymongo import MongoClient
+
+import threading
+
+mongo = MongoClient("mongodb://localhost:27017")
+
+db = mongo["norton"]
+
```

Comment:
It may be a good idea to drop tables if they already exist (to keep things clean!)